### PR TITLE
code samples migration

### PR DESCRIPTION
**Pull request by jonathanlei**
migration for https://xrpl.org/code-samples.html

### CODE REVIEW

**Comment by mDuo13** [comment]
Looking good so far. I guess the links to the repo won't work until after this is merged (since the linked files are part of the PR) but otherwise what's there looks good.

File: _components/CodeSamples.tsx
Comment (JST5000) [inline _components/CodeSamples.tsx:20]: Can you just do this directly as javascript code instead of using `preval`?

File: _components/CodeSamples.tsx
Comment (JST5000) [inline _components/CodeSamples.tsx]: You can delete these if you've used them already :)

File: _components/CodeSamples.tsx
Comment (JST5000) [inline _components/CodeSamples.tsx]: Are you able to choose your language without this?

File: _components/CodeSamples.tsx
Comment (JST5000) [inline _components/CodeSamples.tsx:257]: Did this section let you choose the coding language?

File: _components/CodeSamples.tsx
Comment (JST5000) [inline _components/CodeSamples.tsx:14]: @mDuo13 do you know of a better place to store a variable like this in the file structure?

File: _components/CodeSamples.tsx
Comment (jonathanlei) [reply _components/CodeSamples.tsx:20]: React can't execute fs operations at runtime, this is a workaround

**Comment by mDuo13** [comment]
To fix the build error, you need to run `yarn` and then commit the updated lockfile. I think the lockfile is out of step with the `package.json` because the merge combined two changes to the `package.json` and `yarn` hasn't updated the lockfile for the merged `package.json` file.

**Review by mDuo13 — APPROVED** [review]
This looks good to go now if you want to merge it, @amarantha-k

**Pull request merged**
